Doc/library/sqlite3.rst

```diff
@@ -406,14 +406,22 @@ Connection Objects
 
    .. attribute:: isolation_level
 
-      Get or set the current default isolation level. :const:`None` for autocommit mode or
-      one of "DEFERRED", "IMMEDIATE" or "EXCLUSIVE". See section
-      :ref:`sqlite3-controlling-transactions` for a more detailed explanation.
+      Get or set the current default isolation level.
+      Set to :const:`None` to disable implicit transaction handling,
+      or one of "", "DEFERRED", "IMMEDIATE" or "EXCLUSIVE".
+      Defaults to the former (`""`), unless overridden at :func:`connect`,
+      using the *isolation_level* parameter.
+      Both "" and "DEFERRED" carry the same meaning;
+      they imply deferred isolation level.
+      See :ref:`sqlite3-controlling-transactions` for more details.
 
    .. attribute:: in_transaction
 
+      This read-only attribute corresponds to the low-level SQLite autocommit
+      mode.
```

Comment:
Here, we should either link to the section below that discusses it (if it does), or else directly to SQLite's own autocommit docs.